binary_search/script.js

```diff
@@ -0,0 +1,23 @@
+var sortedNumbers = [1, 2, 3, 4];
+
+console.log(binarySearch(sortedNumbers, 1));
+console.log(binarySearch(sortedNumbers, 2));
+console.log(binarySearch(sortedNumbers, -5));
+console.log(binarySearch(sortedNumbers, 5));
+console.log(binarySearch(sortedNumbers, 2.5));
+
+function binarySearch(arr, key) {
+  var lo = 0, up = arr.length - 1, mid, el;
```

Comment:
I'd suggest to change variable names, it difficult to understand what they mean at the beginning.
also it's better to break line when you initialize variables
```
var lo = 0,
      up = arr.length - 1,
      mid, el;
```
or
```
var lo = 0;
var up = arr.length - 1;
var mid; //var mid, el;
var el;
```
imo the #2 looks better